app/src/pages/inside/dashboardPage/widgets/uniqueBugsTable/messages.jsx

```diff
@@ -0,0 +1,44 @@
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { BUG_ID, SUBMIT_DATE, FOUND_IN, SUBMITTER } from './constants';
+
+const bugIDColumnTitle = (
+  <FormattedMessage id={'UniqueBugsTable.bugIDColumn'} defaultMessage={'BUG ID'} />
+);
+
+const submitDateColumnTitle = (
+  <FormattedMessage id={'UniqueBugsTable.submitDateColumn'} defaultMessage={'SUBMIT DATE'} />
+);
+
+const foundInColumnTitle = (
+  <FormattedMessage id={'UniqueBugsTable.foundInColumn'} defaultMessage={'FOUND IN'} />
+);
+
+const submitterColumnTitle = (
+  <FormattedMessage id={'UniqueBugsTable.submitterColumn'} defaultMessage={'SUBMITTER'} />
+);
+
+export const COLUMN_NAMES_MAP = {
+  [BUG_ID]: {
+    full: bugIDColumnTitle,
+    short: bugIDColumnTitle,
```

Comment:
No need to duplicate titles. If short one is not present, full title is used.

Reply:
Removed